Doc/reference/datamodel.rst

```diff
@@ -1564,6 +1564,17 @@ a module object to a subclass of :class:`types.ModuleType`. For example::
    the module globals (whether by code within the module, or via a reference
    to the module's globals dictionary) is unaffected.
 
+.. versionadded:: 3.5
```

Comment:
Sorry, I missed this earlier. `module.__class__` attribute exists in Python versions older than 3.5. From Python 3.4:

```py
>>> from types import ModuleType
>>> sys.modules['argparse'].__class__
<class 'module'>
>>> sys.modules['argparse'].__class__ = ModuleType
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: __class__ assignment: only for heap types
```

So I think this should be changed to use `versionchanged` instead of `versionadded` and it should say that the `__class__` attribute is now writable.

Reply:
@berkerpeksag  Thanks for catching that!  I've also updated in #5321, which is the backport of only the __class__ doc change to 3.6.